docs/user/tipsandtricks.md

```diff
@@ -143,14 +143,15 @@ In conda-forge, some packages are available with GPU support. These packages not
 In order to override the default behavior, a user can set the environment variable `CONDA_OVERRIDE_CUDA` like below to install TensorFlow with GPU support even on a machine with CPU only.
 
 ```shell-session
-CONDA_OVERRIDE_CUDA="11.2" conda install "tensorflow==2.7.0=cuda112*" -c conda-forge
+CONDA_OVERRIDE_CUDA="<CUDA version>" conda install "tensorflow==2.7.0=cuda<version>*" -c conda-forge
 # OR
-CONDA_OVERRIDE_CUDA="11.2" mamba install "tensorflow==2.7.0=cuda112*" -c conda-forge
+CONDA_OVERRIDE_CUDA="<CUDA version>" mamba install "tensorflow==2.7.0=cuda<version>*" -c conda-forge
```

Comment:
I'm not sure if this is more clear, given the `XX.Y` mapping to `cudaXXY` but I'll defer to other people's thoughts here.